src/app/promoScreen/PromoScreen.js

```diff
@@ -54,7 +53,7 @@ const PromoScreen = () => (
             display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100%', flexDirection: 'column',
           }}
           >
-            <img src={colobus} style={{ marginBottom: 40 }} />
+            <img src={colobus} alt='' style={{ marginBottom: 40 }} />
```

Comment:
https://blog.hubspot.com/marketing/image-alt-text 
What word best represents this image?

Reply:
updated :)

Reply:
Very nice